src/amo/components/AddonReviewCard/index.js

```diff
@@ -258,36 +327,28 @@ export class AddonReviewListItemBase extends React.Component<InternalProps> {
             <a
               href="#edit"
               onClick={this.onClickToEditReview}
-              className="AddonReviewListItem-edit AddonReviewListItem-control"
+              className="AddonReviewCard-edit AddonReviewCard-control"
             >
-              {isReply
-                ? i18n.gettext('Edit my reply')
-                : i18n.gettext('Edit my review')}
+              {this.editPrompt()}
```

Comment:
> It doesn't seem like we should display a link at all in the case of a rating only ... because they don't need the form to edit their rating - they do it by simply clicking on a different star

You and I may know to select a star but it's not totally obvious (there's no prompt, no hint, nothing). This was one of the problems with the current UI that we explicitly set out to fix.

> ...a rating-only review will never show up in the list of reviews, so it doesn't seem like we'd ever need a link for "Edit my rating".

It still affects star rating totals and people are allowed to change their mind so I think they should be able to edit it. If someone leaves a low star rating out of frustration they may have a change of heart later.

Reply:
> You and I may know to select a star but it's not totally obvious (there's no prompt, no hint, nothing). This was one of the problems with the current UI that we explicitly set out to fix.

Fair enough, but I don't really understand what the UI is going to look like when a user does click on the "Edit my rating" button. I guess I need to wait for a future patch to see that.

Reply:
> I don't really understand what the UI is going to look like when a user does click on the "Edit my rating" button.

Ohhh, yeah, that's a good point. Let me file a new issue to work on the simpler UI I proposed. Maybe it only needs a *delete* link.